OAuth/Response/LinkedinUserResponse.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace HWI\Bundle\OAuthBundle\OAuth\Response;
+
+
+class LinkedinUserResponse extends PathUserResponse
+{
+    /**
+     * Helper to extract the preferred locale value from MultiLocaleString
+     * https://docs.microsoft.com/en-us/linkedin/shared/references/v2/object-types#multilocalestring
+     *
+     * @param $path
+     * @return mixed
+     */
+    protected function getPreferredLocaleValue($path)
+    {
+        $multiLocaleString = $this->getValueForPath($path);
```

Comment:
As defined by the documentation, `preferredLocale` is required, thus can be missing.
Moreover, when it is defined, the `country` may be omitted as well, so we'd need to check for that too.

This would take care of both issues:

```php
$multiLocaleString = $this->getValueForPath($path);

$locale = '';
if (isset($multiLocaleString['preferredLocale'])) {
    $locale = $multiLocaleString['preferredLocale']['language'];
    if (!empty($multiLocaleString['preferredLocale']['country'])) {
        $locale .= '_'.$multiLocaleString['preferredLocale']['country'];
    }
}

if (isset($multiLocaleString['localized'][$locale])) {
    return $multiLocaleString['localized'][$locale];
}

$fallbackLocale = array_keys($multiLocaleString['localized'])[0];
return $multiLocaleString['localized'][$fallbackLocale];
```